eventorganizer.api/src/main/java/com/event/organizer/api/model/dto/ProfilePictureRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package com.event.organizer.api.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class ProfilePictureRequestDto {
+    private String url;
+    private Long eventId;
```

Comment:
- We don't need a whole Dto here
- We only need the String url
- We don't need the eventId, we attach the profile picture to an Account, not an Event